app/lib/filters/radio_filter.rb

```diff
@@ -0,0 +1,27 @@
+module Filters
+  class RadioFilter < Filter
```

Comment:
The code here has been copied from TextFilter. Would it make sense to just extend `TextFilter` and not have anything here?